File: apps/zxtune-android/zxtune/src/main/java/app/zxtune/playback/PlayableItem.java
/**
 *
 * @file
 *
 * @brief Playback item extension describing also module
 *
 * @author vitamin.caig@gmail.com
 *
 */

package app.zxtune.playback;

import app.zxtune.core.Module;

public interface PlayableItem extends Item {

  /**
   * @return Associated module
   */
  Module getModule();
}

File: apps/zxtune-android/zxtune/src/main/java/app/zxtune/playback/PlayableItem.kt
/**
 *
 * @file
 *
 * @brief Playback item extension describing also module
 *
 * @author vitamin.caig@gmail.com
 */
package app.zxtune.playback

import app.zxtune.core.Module

interface PlayableItem : Item {
    /**
     * @return Associated module
     */
    val module: Module
}
